doc/nrf/ug_radio_fem.rst

```diff
@@ -566,13 +617,13 @@ Alternatively, add the shield in the project's :file:`CMakeLists.txt` file:
 To build with the |nRFVSC|, specify ``-DSHIELD=nrf21540_ek`` in the **Extra Cmake arguments** field.
 See :ref:`cmake_options`.
 
-When building for a board with an additional network core, for example nRF5340, add an additional ``-DSHIELD`` variable with the *childImageName_* parameter between ``-D`` and ``SHIELD`` to build for the network core as well.
+When building for a board with an additional network core, for example nRF5340, add the ``-DSHIELD`` variable with the *childImageName_* parameter between ``-D`` and ``SHIELD`` to build for the network core. In such case, the application core is only responsible for forwarding pins needed to control the nRF21540 to the network core. It can be set up to do so through setting ``-DSHIELD=nrf21540_ek_fwd``.
```

Comment:
```suggestion
When building for a board with an additional network core, like the nRF5340, add the ``-DSHIELD`` variable with the *childImageName_* parameter between ``-D`` and ``SHIELD`` to build for the network core.
In this case, the application core is only responsible for forwarding to the network core the pins needed to control the nRF21540 .
It can be set up to do so using the ``-DSHIELD=nrf21540_ek_fwd`` setting.
```

It is not that clear what this "It can be set up to do so using the ``-DSHIELD=nrf21540_ek_fwd`` setting.:" does.

Reply:
I clarified this sentence.